### PR TITLE
Fix OSX bug on MachineName #5732.

### PR DESCRIPTION
**Pull request by Priya91**

### CODE REVIEW

File: src/System.Runtime.Extensions/tests/System/Environment.MachineName.cs [hunk 1]
```diff
@@ -25,7 +24,9 @@ internal static string GetComputerName()
             }
             else
             {
-                return Interop.Sys.GetNodeName();
+                string temp = Interop.Sys.GetNodeName();
+                int index = temp.IndexOf('.');
```
Comment (mellinoe) [inline src/System.Runtime.Extensions/tests/System/Environment.MachineName.cs]: Just to make sure, a machine name can't contain a period, right?

**Comment by mellinoe** [comment]
LGTM

**Comment by stephentoub** [comment]
LGTM

File: src/System.Runtime.Extensions/tests/System/Environment.MachineName.cs (hunk 1 above)
Comment (Priya91) [inline src/System.Runtime.Extensions/tests/System/Environment.MachineName.cs]: No, on OSX naming computer name as . stores the computer name as Macintosh.

**Comment by Priya91** [comment]
Test Innerloop OSX Debug Build and Test

**Comment by stephentoub** [comment]
Test Innerloop OSX Debug Build and Test please

**Pull request merged**